src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -133,9 +142,20 @@ public EnumerateDatasetModel enumerate(int offset,
 
     public String delete(String id, AuthenticatedUserRequest userReq) {
         String description = "Delete dataset " + id;
+        CloudPlatform platform;
```

Comment:
can be `final`
```suggestion
        final CloudPlatform platform;
```

Reply:
That doesn't seem to work with try/catch instantiation :(